regression-tests/12-rpl/04-rpl-large-network.csc

```diff
@@ -51,7 +51,7 @@ make sender-node.cooja TARGET=cooja</commands>
       <description>RPL root</description>
       <source>[CONTIKI_DIR]/regression-tests/12-rpl/code/root-node.c</source>
       <commands>make TARGET=cooja clean
-make root-node.cooja TARGET=cooja</commands>
+make root-node.cooja DEFINES=UIP_CONF_MAX_ROUTES=400 TARGET=cooja</commands>

```

Comment:
This is the problematic change: this should not be added here (the test is explicitly set up to have only 8 routes at the root node).